lib/pluginJsonapi.js

```diff
@@ -158,18 +186,38 @@ function pluginJsonapi( server, options, next ) {
           _.each( resourceArray, function( resource ) {
             // build a href for the resource if there's an id property
             if ( resource.id ) {
-              resource.href = _buildHref( request.route.settings.bind.resourceName, resource.id );
+              resource.href = _buildHref( { type: request.route.settings.bind.resourceName, ids: resource.id } );
             }
+
             // If there's no links then there's no actions to take
             if ( resource.links ) {
+
               // Loop each resource in the links and where we have a 'type' and 'id', build a href
               _.each( resource.links, function( value ) {
+                var requestData = {
```

Comment:
I don't think `requestData` is quite the right name here, what is this variable actually for?